pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/application-deployment/index.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: layout.pug
+navigationTitle: Application Deployment
+title: Deployment of Applications
+menuWeight: 5
+beta: false
+excerpt: Deploy applications to attached clusters using the CLI
+---
+
+<!-- markdownlint-disable MD004 MD040 -->
+
+This topic describes how to use the CLI to deploy a workspace catalog application to attached clusters within a workspace.
+
+## Prerequisites
+
+Before you begin, you must have:
+
+- A running cluster with [Kommander installed](../../../../install/).
+- An [existing Kubernetes cluster attached to Kommander](../../../../clusters/attach-cluster/).
+
+Set the `WORKSPACE_NAMESPACE` environment variable to the name of the workspace's namespace where the cluster is attached:
+
+```sh
+export WORKSPACE_NAMESPACE=<workspace_namespace>
+```
+
+## Deploy the application
+
+The list of available applications that can be deployed on the attached cluster can be found [in this documentation](../../catalog-applications#workspace-catalog-applications).
+
+1.  Deploy a supported application to [your existing attached cluster](../../../../clusters/attach-cluster/) with an `AppDeployment` resource.
+
+1.  Within the `AppDeployment`, define the `appRef` to specify which `App` will be deployed:
+
+    ```yaml
+    cat <<EOF | kubectl apply -f -
+    apiVersion: apps.kommander.d2iq.io/v1alpha2
+    kind: AppDeployment
+    metadata:
+      name: cert-manager
+      namespace: ${WORKSPACE_NAMESPACE}
+    spec:
+      appRef:
+        name: cert-manager-0.2.7
```

Comment:
should we use an `App` in this example instead of a `ClusterApp` so it follows more closely with how workspace catalog applications will work? (same with the custom config example below)

Reply:
I don't have a strong opinion here, but consistency is what we strive for in docs, so I say make it consistent with the workspace catalog application.

Reply:
Agree. Updated